crates/autopilot/src/arguments.rs

```diff
@@ -220,6 +220,28 @@ pub struct Arguments {
     /// `order_events` database table.
     #[clap(long, env, default_value = "30d", value_parser = humantime::parse_duration)]
     pub order_events_cleanup_threshold: Duration,
+
+    /// List of addresses which receives a specific discount for the protocol
+    /// fees
+    #[clap(long, env, use_value_delimiter = true)]
+    pub protocol_fee_discounted_addresses: Vec<ProtocolFeeDiscountedAddresses>,
```

Comment:
We don't need an extra type for this. `clap` is able to parse `Vec<H160>` just fine.

Reply:
Let's not stay commented out code around. This could be recovered from github (if you don't rebase it away, I think) or just be rewritten when it's needed because it's not complex.

Reply:
oh my bad, I didn't realize I commented it